systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -22,10 +22,13 @@ public class KeycloakUtils {
 
     private static final Logger LOGGER = LogManager.getLogger(KeycloakUtils.class);
 
+    public final static String PATH_TO_PGO_PREPARE_SCRIPT = "../systemtest/src/test/resources/oauth2/prepare_pg_operator.sh";
     public final static String PATH_TO_KEYCLOAK_PREPARE_SCRIPT = "../systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh";
     public final static String PATH_TO_KEYCLOAK_TEARDOWN_SCRIPT = "../systemtest/src/test/resources/oauth2/teardown_keycloak_operator.sh";
+    public final static String PATH_TO_PGO_TEARDOWN_SCRIPT = "../systemtest/src/test/resources/oauth2/teardown_pg_operator.sh";
 
-    public final static String LATEST_KEYCLOAK_VERSION = "15.0.2";
+
+    public final static String LATEST_KEYCLOAK_VERSION = "18.0.0";
     public final static String OLD_KEYCLOAK_VERSION = "11.0.1";
```

Comment:
Is this needed for 1.16 kube support or we can remove it?

Reply:
I have no idea. Better to ask @see-quick if older version is still needed.

Reply:
Had to make some IFs, for this version, so PG DB is not installed and usage skipped during deployment of KC.